kubemarine/templates/patches/kube-scheduler+json.json.j2

```diff
@@ -0,0 +1,5 @@
+[
```

Comment:
Could keep only one template for control-plane

Reply:
Done. One template for kubelet and one template for all the control-plane pods.